lib/gmath/la.c

```diff
@@ -150,7 +152,7 @@
         return NULL;
     }
 
-    memcpy(&B->vals[0], &A->vals[0], A->cols * A->ldim * sizeof(doublereal));
+    memcpy(&B->vals[0], &A->vals[0], A->cols * A->ldim * sizeof(double));
```

Comment:
I suppose these are newly detected thanks to the change in type. Or maybe re-triggered because of the edit. Anyway, something to fix perhaps in the PR.

Reply:
Of course I'll take a look at that. (When I solved my presently much bigger challenge with configure :-)